src/logic/collectibles/store/reducer/collectibles.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import { type ActionType, handleActions } from 'redux-actions'
+
+import { ADD_NFT_ASSETS, ADD_NFT_TOKENS } from '~/logic/collectibles/store/actions/addCollectibles'
+import type { NFTAssets, NFTToken } from '~/routes/safe/components/Balances/Collectibles/types'
+
+export const NFT_ASSETS_REDUCER_ID = 'nftAssets'
+export const NFT_TOKENS_REDUCER_ID = 'nftTokens'
+
+export type NFTAssetsState = NFTAssets | {}
+export type NFTTokensState = NFTToken[]
+
+export const nftAssetReducer = handleActions<NFTAssetsState, *>(
```

Comment:
I think this can be a single reducer

Reply:
They both have different default values.

I can split them into different files, just for consistency.